wire-ios-transport/Source/PushChannel/StarscreamPushChannel.swift

```diff
@@ -152,7 +155,7 @@ final class StarscreamPushChannel: NSObject, PushChannelType {
         }
         webSocket?.connect()
 
-        Logging.pushChannel.debug("Connecting websocket..")
+        WireLogger.pushChannel.debug("Connecting websocket with URL: \(websocketURL)")
```

Comment:
I guess this will put expose the clientId, let's use attributes instead like:
```
WireLogger.pushChannel.debug("Connecting websocket with URL: \(websocketURL.endpointRemoteLogDescription)", attributes: [.clientId: clientID.redactedAndTruncated()], .safePublic)
```

check RequestLog.swift:81

Reply:
Changed